features/dataset-admin.feature

```diff
@@ -46,6 +45,13 @@ Scenario: form loading with all necessary fields
 	And I should see a button input "Save"
 	And I should see a button "Create New Log"
 
+@ok @issue-381
+Scenario: Admin user can visit admin dataset update page and will not see publisher option
+	Given I sign in as an admin
+	When I am on "/adminDataset/update/id/210"
+	And I should see "Update Dataset 100002"
+	Then I should not see "Publisher"
+
```

Comment:
I think for this issue, we do not need this new test since it is checking for the string `Publisher` in the web page - what would happened if the word `Publisher` is in the dataset description for example? You can just remove `And I should see a form element labelled "Publisher"` from the above scenario which you already have done.

Reply:
The new scenario was removed. Also I should `not` see a form element labelled "Publisher" was implemented and added into the `form loading with all necessary fields` scenario.